kinisi/parser.py

```diff
@@ -122,7 +122,9 @@ def correct_drift(self, disp: VariableLikeType) -> VariableLikeType:
 
         :return: Displacements corrected to account for drift of a framework.
         """
-        return disp - sc.mean(disp['atom', self.drift_indices.values], 'atom')
+
+        nan_replacer = sc.scalar(0.0, unit=self.distance_unit)
```

Comment:
I think I prefer the `if self.drift_indices.size != 0:` solution. Cause this approach might mask other issues -- like if the values are `NaN` for some other reason.